src/ast/passes/unstable_feature.cpp

```diff
@@ -0,0 +1,107 @@
+#include <cstring>
```

Comment:
Optional nit: maybe we want to roll this and the deprecated pass into one "feedback" pass. Both are just providing hints to the user about old or new functionality, so maybe they are logically the same.

I don't actually feel there's a particularly strong reason one way or the other, it's just a logical alternative that I thought I'd mention. Totally your preference.

Reply:
I'm not opposed but let me look at doing that in a follow-up PR.